labs/lab05.ipc/sol/lin/3-funny_sem/client.c

```diff
@@ -0,0 +1,33 @@
+/**
+ * SO
+ * Lab #5
+ *
+ * Task #3, Linux
+ *
+ * Client want's to share a magic number
+ */
+#include <stdio.h>
+#include <fcntl.h>
+#include <sys/types.h>
+#include <semaphore.h>
+
+#include "utils.h"
+
+#define SEM_NAME "/my_sem"
+#define	MAGIC_NUM 42
+
+int main(void)
+{
+	sem_t *sem;
+	int rc;
+
+	printf("Client is sending magic number %d\n", MAGIC_NUM);
+
```

Comment:
```suggestion
/* TODO - Use one semaphore to send MAGIC_NUM to server */

```
to keep the file consistent with `skel/lin/3-funny_sem/client.c`